src/utils.ts

```diff
@@ -390,6 +391,19 @@ export const oauthToken = async (
     worker
   );
 
+  const missingScope = getMissingScope(scope, result.scope);
+  if (missingScope.length) {
+    console.warn(
+      `The requested scopes (${scope}) are different from the scopes part of the retrieved token (${result.scope}). This could result in unexpected behavior and might eventually become unsupported. It is advised to resolve this by either:
+  
+  - Removing \`${missingScope}\` from the scope when requesting a new token.
+  - Ensuring \`${missingScope}\` is returned as part of the requested token's scopes.`
+    );
+  }
```

Comment:
Tried formatting the message so that it shows up like this on the console:

![Uploading image.png…]()

Reply:
Nice